src/qml/filters/audio_adeclick/ui.qml

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) 2023 Meltytech, LLC
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+import QtQuick
+import QtQuick.Controls
+import QtQuick.Layouts
+import Shotcut.Controls as Shotcut
+
+Item {
+    property bool blockControls: false
+
+    function setControls() {
+        if (blockControls)
+            return;
+        sliderWindow.value = filter.getDouble('av.window');
+        sliderThreshold.value = filter.getDouble('av.threshold');
+    }
+
+    width: 200
+    height: 130
+    Component.onCompleted: {
+        if (filter.isNew) {
+            filter.set('av.window', 50);
+            filter.set('av.overlap', 75);
+            filter.set('av.aorder', 2);
+            filter.set('av.threshold', 2);
+            filter.set('av.burst', 2);
+            filter.set('av.method', 'add');
```

Comment:
Per Paul's comment and our version of FFmpeg, I think the method needs to be changed. Did you test with save?

Reply:
Save sounds better with my test clip. So I changed it.